CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 2.8.12)
 # Defer enabling C and CXX languages.
 project(AWSLC NONE)
 
-if(WIN32)
+if(MSVC)
```

Comment:
I didn't understand why is this line changed, could you please clarify?

Reply:
Because cl.exe is Visual Studio compiler and this PR is about when not compiling with Visual Studio.